.circleci/config.yml

```diff
@@ -159,13 +159,20 @@ jobs:
       - attach_workspace:
           # Must be absolute path or relative path from working_directory
           at: .
+      # https://superuser.com/a/1620000/54442
+      - run:
+          name: Install and set Node 16.x
+          command: |
+            $ErrorActionPreference="Stop"
```

Comment:
Are the lines below treated as if they were semicolon-separated?

Reply:
Ish. They are separate commands on seperate lines :)

Reply:
Cool. I wasn't sure if CircleCI had any magic juice it used to run those commands.

Reply:
If you consider PowerShell 5 'magic' 😆